README.md

```diff
@@ -136,6 +136,8 @@ $[1] 127.0.0.1 - - [05/Apr/2024 16:49:39] "GET /static/css/index.css HTTP/1.1" 2
 
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
 
+Note: If you are getting "Insecure Transport Error" errors you can add: `OAUTHLIB_INSECURE_TRANSPORT=1` to your .env file for local dev.
```

Comment:
Probably want this included in your `.env` template file (as it's primarily used for local dev)